client/src/pages/donate-other.js

```diff
@@ -0,0 +1,214 @@
+/* eslint-disable max-len */
+import React, { Component, Fragment } from 'react';
+import Helmet from 'react-helmet';
+import ReactGA from 'react-ga';
+import { Grid, Col, Row } from '@freecodecamp/react-bootstrap';
+
+import Spacer from '../components/helpers/Spacer';
+
+const payPalPayments = [
+  { eventLabel: 'paypal',
+    eventValue: 5,
+    defaultValueHash: 'KTAXU8B4MYAT8',
+    defaultValue: 'Donate $5 each month'
+  },
+  { eventLabel: 'paypal',
+    eventValue: 10,
+    defaultValueHash: 'BYEBQEHS5LHNC',
+    defaultValue: 'Donate $10 each month'
+  },
+  { eventLabel: 'paypal',
+    eventValue: 35,
+    defaultValueHash: '57ZPMKJ8G893Y',
+    defaultValue: 'Donate $35 each month'
+  },
+  { eventLabel: 'paypal',
+    eventValue: 50,
+    defaultValueHash: '2ZVYTHK6Q7AFW',
+    defaultValue: 'Donate $50 each month'
+  },
+  { eventLabel: 'paypal',
+    eventValue: 100,
+    defaultValueHash: 'C7PUT3LMJHKK2',
+    defaultValue: 'Donate $100 each month'
+  },
+  { eventLabel: 'paypal',
+    eventValue: 250,
+    defaultValueHash: '69JGTY4DHSTEN',
+    defaultValue: 'Donate $250 each month'
+  },
+  { eventLabel: 'paypal one time donation',
+    eventValue: 0,
+    defaultValueHash: 'B256JC6ZCWD3J',
+    defaultValue: 'Make a one-time donation'
+  }
+];
+
+class IndexPage extends Component {
```

Comment:
This is no longer the `IndexPage`. It should be `DonateOtherPage`

Reply:
All pages in the client app are called `IndexPage`. I just assumed it was convention?

Reply:
hmmm... I must have missed it in my hasty refactor. I will raise a PR to cover the others.